corehq/apps/locations/adjacencylist.py

```diff
@@ -76,13 +76,14 @@ def make_cte_query(cte):
             .order_by(("" if ascending else "-") + "_depth")
         )
 
-    def get_descendants(self, node, include_self=False):
+    def get_descendants(self, node, include_self=False, **filters):
```

Comment:
I'm assuming you tried `...get_descendants(...).filter(**filters)` and that didn't work. Was it something with the duplicate removal that made this necessary? If not, I'm not sure this is necessary since the return value of this method is a query object on which you can use `.filter(...)`

Reply:
@millerdev Mmm I can't remember now why I put the filters so deep down...then again I changed this code so many times...but you're right, it seems redundant now, so I'll put it higher upstream. The tests don't seem to mind...